tools/spec-gen-sdk/src/automation/reportStatus.ts

```diff
@@ -85,6 +85,7 @@ export const generateReport = (context: WorkflowContext) => {
 
   executionReport = {
     packages: packageReports,
+    services: context.services,
```

Comment:
Could we have service name added as part of the package object so that we won't be bothered on their mapping?

Reply:
Am not sure what you mean by this